.github/workflows/release.yml

```diff
@@ -65,10 +65,10 @@ jobs:
           git push origin HEAD:main
         if: inputs.draft != true
 
-      - name: Set up Ruby 3
+      - name: Set up Ruby
         uses: ruby/setup-ruby@v1
         with:
-          ruby-version: '3.2.0'
+          ruby-version: ruby
```

Comment:
should this explicitly select a version? or does this just pick the latest release?